src/scss/_page-controls.scss

```diff
@@ -1,48 +1,48 @@
-.page-controls {
-    .previous, .next {
-        position: fixed;
-        left: calc(50% + #{$navbar-width/2});
-        transform: translate(-50%, 0);
-        z-index: 2;
-
-        padding: 5px 10px;
-        font-size: 40px;
-        font-weight: 100;
-
-        &:hover {
-            cursor: pointer;
-        }
-
-        display: none;
-        &.active {
-            display: block;
-        }
-    }
-
-    .previous {
-        top: 15px;
-    }
-    .next {
-        bottom: 15px;
-
-        transition: color ease 0.5s;
-        &.white {
-            color: white;
-        }
-    }
-}
-
-@media #{$tablet-media-query} {
-    .page-controls {
-        .previous, .next {
-            left: calc(50% + #{$navbar-mobile-width/2});
-        }
-    }
-}
-
-@media #{$phone-media-query} {
-    .page-controls {
-        display: none;
-    }
-
-}
+// .page-controls {
```

Comment:
It is ok to comment all page controls related code, and I'd leave it that way. I just wanted to point out 2 other solutions you could have used:
* Instead of commenting the actual logic and code of that component, you could have just commented of deleted the imports of these components, i.e.:
  * In ```layouts/index.html``` comment/delete ```{{ partial "pageControls.html" . }}```
  * In ```src/scss/style.scss``` comment/delete ```@import 'page-controls';```
* You could just delete everything related to page controls: html, css and imports. The thing with repositories is that we have history so if we ever need the controls again, we can just go in history and put it back. This way, no dead code left in the source, which is usually a better practice.